website/content/api-docs/system/auth.mdx

```diff
@@ -99,6 +133,11 @@ For example, enable the "foo" auth method will make it accessible at
   - `allowed_response_headers` `(array: [])` - List of headers to whitelist,
     allowing a plugin to include them in the response.
 
+  - `plugin_version` `(string: "")` – Specifies the semantic version of the plugin
+    to use, e.g. "v1.0.0". If unspecified, the server will select any matching
+    unversioned plugin that may have been registered, the latest versioned plugin
+    registered, or a builtin plugin in that order of precendence.
```

Comment:
I just want to clarify if the `plugin_version` parameter in this location is meant to be one of the sub-list values of the `config` parameter's list above? (Starting line 111).

Reply:
Correct 👍 it's because it needs to be in the `config` to be tuneable after the initial mount.